app/src/components/dcCard/DCCard.tsx

```diff
@@ -0,0 +1,117 @@
+"use client";
+import React, { RefObject } from "react";
+import {
+  Card,
+  CardHeading,
+  Text,
+  CardContent,
+  Tooltip,
+  StatusBadge,
+} from "@nypl/design-system-react-components";
+import styles from "./Card.module.css";
+import { headerBreakpoints } from "../../utils/breakpoints";
+import { CollectionCardDataType } from "../../types/CollectionCardDataType";
+import { TRUNCATED_LENGTH } from "@/src/config/constants";
+import ItemCardDataType from "@/src/types/ItemCardDataType";
+interface DCCardProps {
+  cardOffset?: number[];
+  cardRef?: RefObject<HTMLDivElement>;
+  id: string;
+  isLargerThanLargeTablet: boolean;
+  slug?: string;
+  record: CollectionCardDataType | ItemCardDataType;
+}
+
+function isCollectionCardDataType(
+  record: CollectionCardDataType | ItemCardDataType
+): record is CollectionCardDataType {
+  return "numberOfDigitizedItems" in record;
+}
+
+const DCCard = ({
+  cardOffset,
+  cardRef,
```

Comment:
Can you rename this as `ref` or use React's `forwardRef` function https://react.dev/reference/react/forwardRef?